packages/@ember/-internals/glimmer/lib/helpers/array.ts

```diff
@@ -0,0 +1,37 @@
+import { Arguments, VM } from '@glimmer/runtime';
+
+/**
+@module ember
+*/
+
+/**
+   Use the `{{array}}` helper to create an array to pass as an option to your
+   components.
+
+   ```handlebars
+   {{my-component people=(array
+     'Tom Dade'
+     'Yehuda Katz'
+     myOtherPerson)
```

Comment:
Lets use `this.myOtherPerson`

Reply:
I was wondering about that. Should the templates in the test also refer to `this` where there is a backing component? Also the `this.render` calls, are the second parameter considered arguments? I could change the tests to use `this.` and also `@`.